Updated waf-sentinel.md with new section of Automatically detect and respond to threats

shabaz-github opened the pull request:
Updated waf-sentinel.md with new section of Automatically detect and respond to threats

shabaz-github committed:
Added image for Waf-detections

Added image for Waf-detections

shabaz-github committed:
Update waf-sentinel.md

Updated text with New section 'Automatically detect and respond to threats'

Miskatonic-Electronic commented on articles/web-application-firewall/waf-sentinel.md:
```diff
@@ -79,6 +79,17 @@ To enable log analytics for each resource, go to your individual Azure Front Doo
+   
+## Automatically detect and respond to threats
+
+Using Sentinel ingested WAF logs, you can use Sentinel analytics rules to automatically detect security attacks, create security incident, and automatically respond to security incident using playbooks. Learn more [Use playbooks with automation rules in Microsoft Sentinel](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fazure%2Fsentinel%2Ftutorial-r ...
```
I think your URL should be updated to not include the outlook safe link.

https://learn.microsoft.com/azure/sentinel/tutorial-respond-threats-playbook?tabs=LAC

shabaz-github committed:
Updated with right Link for Use Playbooks

Updated with right Link for Use Playbooks with automation rules in Microsoft Sentinel

Jak-MS commented on articles/web-application-firewall/waf-sentinel.md:
```diff
@@ -79,6 +79,17 @@ To enable log analytics for each resource, go to your individual Azure Front Doo
+   
+## Automatically detect and respond to threats
+
+Using Sentinel ingested WAF logs, you can use Sentinel analytics rules to automatically detect security attacks, create security incident, and automatically respond to security incident using playbooks. Learn more [Use playbooks with automation rules in Microsoft Sentinel](https://learn.microsoft.com/azure/sentinel/tutorial-respond-threats-playbook?tabs=LAC).
```
```suggestion
Using Sentinel ingested WAF logs, you can use Sentinel analytics rules to automatically detect security attacks, create security incident, and automatically respond to security incident using playbooks. Learn more [Use playbooks with automation rules in Microsoft Sentinel](../sentinel/tutorial-respond-threats-playbook.md?tabs=LAC).
```
changing to a relative link

Jak-MS committed:
Update articles/web-application-firewall/waf-sentinel.md
